vendor/assets/javascripts/neat/collection_editor.coffee

```diff
@@ -142,7 +143,7 @@ class window.Neat.CollectionEditor extends Backbone.View
       @views.push(view)
 
       $el = $(view.render().el)
-      $el.toggleClass 'alt', !(alt = !alt)
+      $el.toggleClass 'alt', !(alt = !alt) if @alternateRows
```

Comment:
This was actually causing issues with styling some of the elements contained in an `alt` line. For example, if you had `.alt > .bubble`. CSS could be used to un-style these descendent elements, but there would be a lot less to counteract if the class wasn't added in the first place.